### ISSUE TITLE
network\data\twitter_mutual2.json,How to calculate the node coordinates (node.x, node.y) to avoid duplication of node coordinates

### ISSUE THREAD

**Issue by MengmengWang** [oxfordinternetinstitute/InteractiveVis#60]
(My English is very poor.write by translation tool) .I would like to ask how the node coordinates (node.x, node.y) of twitter_mutual2.json are calculated.
Would like to reply to me, thank you very much,
E-mail:wangmeng6447@163.com

**Comment by computermacgyver** [comment]
Hi Mengmeng,

We used Gephi (http://www.gephi.org/) to layout the network. I can't remember well, but we either used the "Force Atlas 2" or "Fruchterman Reingold" algorithms in Gephi. 

We first deleted non-mutual edges. That is, we deleted an edge if A followed B, but B did not follow A. All the remaining edges are mutual (A follows B and B follows A).

Finally, we used the sigma exporter plugin in Gephi to create the JSON file.

Hope this helps!
